crates/primitives/src/alloy_compat.rs

```diff
@@ -36,7 +36,13 @@ impl TryFrom<alloy_rpc_types::Block> for Block {
                     .collect(),
                 alloy_rpc_types::BlockTransactions::Hashes(_) |
                 alloy_rpc_types::BlockTransactions::Uncle => {
-                    return Err(ConversionError::MissingFullTransactions)
+                    // alloy deserializes empty blocks into `BlockTransactions::Uncle`, if the tx
```

Comment:
I don't think this note is correct, [] would be `Hashes` because it's the first variant

Reply:
right, thought it was uncle because of `default`, but it's not